### ISSUE TITLE
Changing Fragment object annotation style

### ISSUE THREAD

**Issue by daccachejoe** [timoast/signac#1370]
Hi-

I know this issue has come up in the past when people's data is not annotated in the same way as their `annotation` objects are and they have had to alter the `annotation` object via `seqlevelsStyle`. I encountered an issue combining two datasets who's `fragments.tsv.gz` files were annotated differently and as such the `Fragment` object would not align. I ended up working around this by going back and editing the `fragments.tsz.gz` files of one dataset to match the other but had to do so in linux and re-index the file. If possible, I think it would be incredibly helpful to call a function similar to `seqlevelsStyle` on the `Fragments` object and alter the annotation style that way. Not sure if this is something within the package already that I missed though....

Thanks!

**Comment by timoast** [comment]
Agree, it would be useful and should be possible to convert seqlevel style upon reading from the fragment file, especially in cases where different fragment files use different styles. Ability to set a seqlevelStyle for the object itself will be a longer-term goal as it will take some work to implement.

**Comment by timoast** [comment]
This has now been added in the v2 branch
